exercises/concept/meltdown-mitigation/.docs/instructions.md

```diff
@@ -0,0 +1,72 @@
+# Instructions
+
+In this exercise, we'll develop a simple control system for a nuclear reactor.
+
+For a reactor to produce the power it must be in a state of _criticality_.
+If the reactor is in a state less than criticality, it can become damaged.
+If the recator state goes beyond criticality, it can overload and result in a meltdown.
```

Comment:
nit: minor type for reactor

Reply:
My spelling is often the most creative thing about me.  😉  😆